src/main.js

```diff
@@ -83,27 +137,102 @@ function addTask() {
      * @todo 1. Send the request to add the task to the backend server.
      * @todo 2. Add the task in the dom.
      */
+    
+    const To_do_task = document.querySelector("#inputTask").value.trim();
+
+    if (To_do_task === '') {
+        displayErrorToast("Enter the field required..");
+        return;
+    }
+
+    displayInfoToast("Please wait...");
+
+    const dataForApiRequest = {
+        title: To_do_task
+    }
+    const headersForApiRequest = {
+        Authorization: 'Token ' + localStorage.getItem('token')
+    }
+
+    axios({
+        headers: headersForApiRequest,
+        url: API_BASE_URL + 'todo/create/',
+        method: 'POST',
+        data: dataForApiRequest,
+    }).then(function ({ data, status }) {
+        displaySuccessToast("Task has been added !!!");
+        document.querySelector("#inputTask").value = '';
+        getTasks();
+    }).catch(function (err) {
+        displayErrorToast("Task could not be added ...Please try again !!");
+    })
 }
 
-function editTask(id) {
+export function editTask(id) {
     document.getElementById('task-' + id).classList.add('hideme');
     document.getElementById('task-actions-' + id).classList.add('hideme');
     document.getElementById('input-button-' + id).classList.remove('hideme');
```

Comment:
While editing, the input box at first should display the current value of the task instead of an empty text box.